vignettes/cmdstanr-internals.Rmd

```diff
@@ -101,9 +101,15 @@ mod$compile()
 mod$exe_file()
 ```
 
-In all cases the executable is created in the same directory as the file
-containing the Stan program.
+### Executable location
 
+By default, the executable is created in the same directory as the file
+containing the Stan program. You can also specify a different location with the 
+`dir` argument of the `cmdstan_model` function.
+
+```{r compile-method, eval = FALSE}
+mod <- cmdstan_model(stan_file, dir = "path/to/directory/for/executable")
+```
```

Comment:
There's already a chunk named `compile-method` so knitting this vignette gives the error: 

```
processing file: cmdstanr-internals.Rmd
Error in parse_block(g[-1], g[1], params.src) : 
  duplicate label 'compile-method'
```

Just needs a new chunk name like `compile-with-dir`